front/src/Components/EventsTab.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
```

Comment:
./src/Components/EventsTab.js
  Line 1:  React.PropTypes is deprecated since React 15.5.0, use the npm module prop-types instead  react/no-deprecated